.travis.yml

```diff
@@ -10,16 +10,8 @@ matrix:
   - python: "3.6"
     env: TEST_SUITE=lint
 install:
-    # Install codecov, the library for the code coverage reporting tool we use
-  - pip install codecov
-
-    # Install pycodestyle, the library for checking pep8 code conformity.
-  - pip install pycodestyle==2.3.1
-
-    # Install all API packages
-  - pip install ./zulip
-  - pip install ./zulip_bots
-  - pip install ./zulip_botserver
+  - tools/provision
+  - source "zulip-api-py2-venv/bin/activate"
```

Comment:
Ahh, so `python-coverage` is the library for that, which would be reasonable to add to a development-only version of `requirements.txt`.  But `codecov`, the library for talking to the CodeCov server (Which probably declares a dependency on `python-coverage`), we don't need except in Travis CI.